backend/src/test/java/com/woowacourse/momo/auth/controller/AuthControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package com.woowacourse.momo.auth.controller;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.woowacourse.momo.auth.dto.LoginRequest;
+import com.woowacourse.momo.member.dto.request.SignUpRequest;
+import com.woowacourse.momo.member.service.MemberService;
+
+@Transactional
+@AutoConfigureMockMvc
+@SpringBootTest
+class AuthControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private MemberService memberService;
+
+    @BeforeEach
+    void setUp() {
+        SignUpRequest request = new SignUpRequest("woowa@woowa.com", "Woowa1!", "모모");
+        memberService.signUp(request);
+    }
+
+    @DisplayName("정상적으로 로그인될 시 토큰이 발급된다.")
```

Comment:
```suggestion
    @DisplayName("정상적으로 로그인될 시 토큰이 발급된다")
```
마지막에 `온점` 을 붙이거나 안붙이거나 하나로 통일하는게 좋을것 같습니다.